packages/app/vite.config.mjs

```diff
@@ -46,23 +46,27 @@ const config = makeConfig({
   ],
 })
 
-// With some trial and error, it appears that the module federation plugin needs to be added 
-// to the plugins array first so that the dynamic modules are available properly with respect 
+// With some trial and error, it appears that the module federation plugin needs to be added
+// to the plugins array first so that the dynamic modules are available properly with respect
 // to the other plugins.
-config.plugins.unshift(
-  ...federation({
-    name: 'host',
-    remotes: {
-      'app-studio': {
-        type: 'module',
-        name: 'app-studio',
-        entryGlobalName: 'app-studio',
-        entry: '/__cypress-studio/app-studio.js',
-        shareScope: 'default',
-      },
-    },
-    filename: 'assets/app-studio.js',
-  }),
-)
+// config.plugins.unshift(
```

Comment:
whats the reason we are commenting this out?

Reply:
Oh yeah forgot to remove this. It's not necessary anymore

Reply:
Removed: [`6fb9d30` (#31129)](https://github.com/cypress-io/cypress/pull/31129/commits/6fb9d30536f2a5e1c9575a5b030525a1050e36ef)